### PR TITLE
CFY-7608 new amqpinflux RPM

### PR DESCRIPTION
**Pull request by funkyHat**

### CODE REVIEW

**Comment by funkyHat** [comment]
Something's up with the github branch protection settings here?

File: packaging/cloudify-amqp-influx.spec [hunk 1]
```diff
@@ -0,0 +1,67 @@
+
+Name:           cloudify-amqp-influx
+Version:        %{CLOUDIFY_VERSION}
+Release:        %{CLOUDIFY_PACKAGE_RELEASE}%{?dist}
+Summary:        Cloudify's REST Service
+Group:          Applications/Multimedia
+License:        Apache 2.0
+URL:            https://github.com/cloudify-cosmo/cloudify-amqp-influxdb
+Vendor:         Gigaspaces Inc.
```
Comment (tehasdf) [inline packaging/cloudify-amqp-influx.spec]: cloudify platform ltd

File: packaging/cloudify-amqp-influx.spec [hunk 2]
```diff
@@ -0,0 +1,67 @@
+
+Name:           cloudify-amqp-influx
+Version:        %{CLOUDIFY_VERSION}
+Release:        %{CLOUDIFY_PACKAGE_RELEASE}%{?dist}
+Summary:        Cloudify's REST Service
```
Comment (tehasdf) [inline packaging/cloudify-amqp-influx.spec]: not restservice

File: packaging/cloudify-amqp-influx.spec [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+
+%define _venv /opt/amqpinflux/env
+
+
+%description
+Cloudify's Broker pulls Cloudify formatted Metrics from RabbitMQ and posts them in InfluxDB.
+
+
+%prep
```
Comment (tehasdf) [inline packaging/cloudify-amqp-influx.spec]: remove empty blox

File: packaging/cloudify-amqp-influx.spec [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+# Copy static files into place. In order to have files in /packaging/files
+# actually included in the RPM, they must have an entry in the %files
+# section of this spec file.
+cp -R ${RPM_SOURCE_DIR}/packaging/files/* %{buildroot}
+
+
+%pre
+
+groupadd -fr amqpinflux
+getent passwd amqpinflux >/dev/null || useradd -r -g amqpinflux -d /opt/amqpinflux -s /sbin/nologin amqpinflux
```
Comment (tehasdf) [inline packaging/cloudify-amqp-influx.spec]: this doesn't need cfyuser anywhere? what about reading the ca cert?

File: packaging/Vagrantfile [hunk 5]
```diff
@@ -0,0 +1,103 @@
+########
```
Comment (tehasdf) [inline packaging/Vagrantfile]: remove the vagrantfile again?

File: .circleci/config.yml [hunk 6]
```diff
@@ -19,7 +19,7 @@ jobs:
             git checkout -qf FETCH_HEAD
           fi
 
-      - run: if [ -n \"$INFLUXDB_VERSION\" ] ; then ./install_influxdb.sh \"$INFLUXDB_VERSION\"; fi
+      - run: if [ -n "$INFLUXDB_VERSION" ] ; then ./install_influxdb.sh "$INFLUXDB_VERSION"; fi
```
Comment (tehasdf) [inline .circleci/config.yml]: do we actually need the if? the env var is hardcoded to always be there anyway..

File: packaging/files/usr/lib/systemd/system/cloudify-amqpinflux.service [hunk 7]
```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Cloudify AMQP InfluxDB Broker Service
+Wants=cloudify-influxdb.service cloudify-rabbitmq.service
+After=cloudify-influxdb.service cloudify-rabbitmq.service
+
+[Service]
+TimeoutStartSec=0
+Restart=on-failure
+EnvironmentFile=-/etc/sysconfig/cloudify-amqpinflux
```
Comment (tehasdf) [inline packaging/files/usr/lib/systemd/system/cloudify-amqpinflux.service]: not optional

**Comment by tehasdf** [comment]
please see the "outdated" comments in my review as well

File: packaging/cloudify-amqp-influx.spec (hunk 4 above)
Comment (funkyHat) [reply packaging/cloudify-amqp-influx.spec]: The behaviour is copied from https://github.com/cloudify-cosmo/cloudify-manager-install/pull/98/files#diff-073ab67cc715e5ab82d7764a190f7fa4L58 which doesn't seem to use `cfyuser`

File: .circleci/config.yml (hunk 6 above)
Comment (funkyHat) [reply .circleci/config.yml]: good point

**Pull request merged**